otlp/logs.go

```diff
@@ -56,13 +57,10 @@ func TranslateLogsRequest(request *collectorLogs.ExportLogsServiceRequest, ri Re
 					attrs["severity_text"] = log.SeverityText
 				}
 				if log.Body != nil {
-					if val, truncatedBytes := getValue(log.Body); val != nil {
-						attrs["body"] = val
-						if truncatedBytes != 0 {
-							// if we trim the body, add telemetry about it
-							attrs["meta.truncated_bytes"] = truncatedBytes
-							attrs["meta.truncated_field"] = "body"
-						}
+					if truncatedBytes := flatten(attrs, options.bodyPrefix, log.Body, 0, options); truncatedBytes != 0 {
```

Comment:
This will result in new log bodies for customers.  Instead of the json string we'd create from the map, they'll get new fields on the event prefixed with `body`.  The end result, I believe, will be an event without a field named `body`.  That doesn't feel quite right to me.

We could leave the existing body conversion logic intact and do the new flatten logic in addition.  Would that be beneficial to customers? It would mean 1 less breaking change.

Reply:
Correct, this currently replaces the single event field `body` with multiple event fields composed of the attributes found in the log body value.

If the log body attribute was a string, it would result in a single event field called `body`.

Yeah - I could add options to allow additive changes so we can enable/disable at our discretion 👍🏻